sycl/include/CL/sycl/handler.hpp

```diff
@@ -313,8 +314,7 @@ class __SYCL_EXPORT handler {
   ///
   /// \param Queue is a SYCL queue.
   /// \param IsHost indicates if this handler is created for SYCL host device.
-  handler(shared_ptr_class<detail::queue_impl> Queue, bool IsHost)
-      : MQueue(std::move(Queue)), MIsHost(IsHost) {}
+  handler(const shared_ptr_class<detail::queue_impl> &Queue, bool IsHost);
```

Comment:
NIT: SYCL2020 deprecates C++ std lib class aliases, such as shared_ptr_class, also std::shared_ptr is used in other RT places. So I `const std::shared_ptr<detail::queue_impl>` would be nicer, since Queue type is being changed anyway.

Reply:
If we need to do it, it should be done in a lot of places. Is it OK if we do in a separate patch?